server/validation/validation.go

```diff
@@ -1,76 +1,303 @@
 package validation
```

Comment:
Thinking about moving this package. It doesn't really belong in `server/`. I think it makes more sense for this to belong to the `tuple` package. @craigpastro @miparnisari any recommendations?

Reply:
Either tuple or in the validations package itself. I would probably prefer the latter.

Reply:
Ok I've moved this into `internal/validation`.